### DISCUSSION TITLE
How to reset or delete world claims?

### DISCUSSION THREAD

**Discussion by eliyash31** [GriefPrevention/GriefPrevention#989] [Q&A] [answered]
Hello how can I reset or delete all the world claims? or can I delete all the claims of players on a specific world?
Since our server will reset the world every 6 months or every Season, I need to know how to delete claims on all worlds.

Mysql is a little bit buggy this time so I can't use it , also no one was yet to reply on the issue I posted about the bonus claims automatically resets back to 0 everyday. so maybe it's best to just the local data for now

**Comment by Jikoo** [comment, marked as answer]
`/deleteclaimsinworld <world>`

**Comment by eliyash31** [comment]
Okay thanks :)
